.travis.yml

```diff
@@ -20,6 +20,7 @@ script:
 
 env:
   global:
+    - DISABLE_SOURCE_MAPS=true
```

Comment:
This ensures that Travis builds do not contain sourcemaps. We will remove this in the future when all kinks in the ember-cli sourcemap pipeline are ironed out.